code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -92,6 +92,13 @@
 /turf/open/return_analyzable_air()
 	return return_air()
 
+/turf/open/proc/has_gas(datum/gas, mole_amount)
+	if(air && air.total_moles())
+		var/loc_gases = air.gases
+		if(loc_gases[gas] && loc_gases[gas][MOLES] >= mole_amount)
+			return TRUE
```

Comment:
I'd prefer ASSERT_GAS to be used here, to maintain consistency with other uses of gaslists.

Reply:
This was added to the `has_gas` proc.